plugin_template.rb

```diff
@@ -361,14 +345,69 @@ def source_paths
   test/dummy/db/*.sqlite3
   test/dummy/db/*.sqlite3-journal
   node_modules
-  package.json
   yarn.lock
+  yarn-error.log
+  package-lock.json
 CODE
 
 #
-# Remove MIT License
+# Replace License
 #
 remove_file 'MIT-LICENSE'
+create_file 'LICENSE', <<~CODE
```

Comment:
wouldn't it be better to read the file from the gem source so we don't need to ever worry about keeping this up to date should anything with the license change?

Reply:
Spoke with @mttdffy about this one. Since I'm trying to push people into using the version of this file publicly available on our most recent stable branch (as documented in the introductory comment in this file) to make sure they're getting the most up-to-date version of the template for the current version of Workarea without having to first clone the Workarea repo, reading the file remotely won't work. Plus, the likelihood of this file changing is pretty low.